### PR TITLE
Replace deprecated kube labels

### PR DESCRIPTION
**Pull request by mburke5678**
Replace deprecated kube labels

### CODE REVIEW

**Comment by mburke5678** [comment]
@kasturinarra @elmiko You helped @jeana-redhat with a PR announcing the deprecation of a few  Kubernetes labels. Would you be willing/able to review this PR that replaces them in the docs?

**Comment by kasturinarra** [comment]
@mburke5678 sure, will get the review done today, thanks !!

**Review by elmiko — COMMENTED** [review]
this looks accurate to me

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -1264,7 +1264,7 @@ EOF
 ----
 Name:               node1.example.com
 Roles:              worker
-Labels:             beta.kubernetes.io/arch=amd64
+Labels:             kubernetes.io/region=amd64
```
Comment (kasturinarra) [inline contributing_to_docs/doc_guidelines.adoc:1267]: should not this be `kubernetes.io/arch=amd64` ?

https://kubernetes.io/docs/reference/labels-annotations-taints/#beta-kubernetes-io-arch-deprecated -> source

File: modules/nodes-nodes-viewing-listing.adoc [hunk 2]
```diff
@@ -106,13 +106,17 @@ $ oc describe node node1.example.com
 ----
 Name:               node1.example.com <1>
 Roles:              worker <2>
-Labels:             beta.kubernetes.io/arch=amd64   <3>
-                    beta.kubernetes.io/instance-type=m4.large
+Labels:             beta.kubernetes.io/os=linux
```
Comment (kasturinarra) [inline modules/nodes-nodes-viewing-listing.adoc:109]: Not sure if this is valid, we will need to change this to `kubernetes.io/os`

https://v1-26.docs.kubernetes.io/docs/reference/labels-annotations-taints/#beta-kubernetes-io-os-deprecated -> source link

File: modules/nodes-nodes-viewing-listing.adoc [hunk 3]
```diff
@@ -106,13 +106,17 @@ $ oc describe node node1.example.com
...
+                    kubernetes.io/hostname=ip-10-0-131-14
                     kubernetes.io/os=linux
-                    failure-domain.beta.kubernetes.io/region=us-east-2
-                    failure-domain.beta.kubernetes.io/zone=us-east-2a
-                    kubernetes.io/hostname=ip-10-0-140-16
+                    kubernetes.io/region=amd64 <3>
```
Comment (kasturinarra) [inline modules/nodes-nodes-viewing-listing.adoc:112]: This should be kubernetes.io/arch=amd64

File: modules/nodes-scheduler-node-selectors-about.adoc [hunk 4]
```diff
@@ -52,14 +52,14 @@ metadata:
...
     node.openshift.io/os_version: '4.5'
     node-role.kubernetes.io/worker: ''
-    failure-domain.beta.kubernetes.io/region: us-east-1
+    topology.kubernetes.io/region: us-east-1
     node.openshift.io/os_id: rhcos
-    beta.kubernetes.io/instance-type: m4.large
+    node.kubernetes.io/instance-type: m4.large
     kubernetes.io/hostname: ip-10-0-131-14
-    beta.kubernetes.io/arch: amd64
+    kubernetes.io/region: amd64
```
Comment (kasturinarra) [inline modules/nodes-scheduler-node-selectors-about.adoc:62]: instead of region it should be arch

**Comment by kasturinarra** [comment]
@mburke5678 reviewed. Could you please help check ? thanks !!

**Review by sjhala-ccs — COMMENTED** [review]
LGTM

**Pull request merged**